public/index.html

```diff
@@ -6,17 +6,39 @@
     <link
       rel="stylesheet"
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
-    />
+      />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Make an appointment to get your COVID vaccine"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+      content="Make an appointment to get your Massachusetts COVID vaccine"
+      />
+
+    <meta name="keywords"
+          content="massachusetts, mass, ma, covid, covid-19, vaccine, vaccines, vax, search, sign up, zipcode"
+          />
+    <meta name="robots" content="index, follow" />
+    <meta name="language" content="English" />
+    <meta name="title" content="Covid-19 Vaccine Appointments in Massachusetts | MACovidVaccines.com" />
```

Comment:
Can we change all instances of "Covid" to "COVID" please? Hope you don't mind throwing that in here as well

Reply:
(Except MACovidVaccines)

Reply:
Fixed